src/Server/Player/PlayerLogic.py

```diff
@@ -102,6 +104,10 @@ def _finish_attack(self):
 
     def restore_attack_ability(self):
         self._attack_on_cooldown = False
+        if self.id in Server.get_instance().get_client_ids():
+            msg = BaseMessage(mess_type=sermess.MessageType.ATTACK_ABILITY, target=sermess.Target.SCREEN)
+            msg.value = True
+            Server.get_instance().send_message(msg, self.id)
```

Comment:
You could make a function out of this. These lines are the same, only value changes.